.env.docker

```
@@ -0,0 +1,70 @@
+# Docker Compose environment for local / self-hosted evaluation.
+# All values work out of the box with docker-compose.images.yaml.
+# Review secrets before any non-local deployment.
+
+# ── Postgres ──────────────────────────────────────────────────────────
+DATABASE_HOST=postgres
+DATABASE_READ_ONLY_HOST=postgres
+DATABASE_PORT=5432
+DATABASE_NAME=postgres
+DATABASE_USER=postgres
+DATABASE_PASSWORD=postgres123
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATABASE_PASSWORD key should go before the DATABASE_PORT key

(UnorderedKey)